ecosystem/sep-0038.md

```diff
@@ -0,0 +1,176 @@
+`currency` | string | (optional) The [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) code of the currency you want to get the price of available Stellar assets in.
+
+#### Response
+
+Only one of the following attributes should be included in the response.
+
+Name | Type | Description
+-----|------|------------
+`assets` | array | (optional) An array of objects containing the available Stellar assets and their prices in terms of the `currency` provided in the request.
+`currencies` | array | (optional) An array of objects containing the available currencies and their prices in terms of the `asset` provided in the request.
+
+`assets` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`asset` | string | Same as the definition of `asset` in the request.
+`price` | string | The price of one unit of `asset` in terms the `currency` sent in the request.
+`decimals` | integer | The number of decimals needed to represent asset.
+
+`currencies` Object Schema
+
+Name | Type | Description
+-----|------|------------
+`currency` | string | Same as the definition of `currency` in the request.
+`price` | string | The price of one unit of `currency` in terms of the `asset` sent in the request.
+`decimals` | integer | The number of decimals needed to represent the currency.
+
+### GET Price
+
+This endpoint can be used to fetch the indicative price of a Stellar asset and currency [pair](https://en.wikipedia.org/wiki/Currency_pair).
+
+#### Request
+
+Name | Type | Description
+-----|------|------------
+`base` | string | The base asset or currency. Ex. `USDC:circle.io`, `ARS`
+`counter` | string | The counter asset or currency.
+`base_amount` | string | The amount of the base asset or currency to be traded for the counter asset or currency.
+
+#### Response
+
+Name | Type | Description
+-----|------|------------
+`price` | string | The price of one unit of the base asset or currency in terms of the counter asset or currency for the `base_amount` specified in the request.
+
+### POST Quote
+
+This endpoint can be used to request a [firm](https://www.investopedia.com/terms/f/firmquote.asp) quote for a Stellar asset and currency pair. 
+
+In contrast with the `GET price(s)` endpoints, the amount requested must be held in reserve and not used in calculations of subsequent quotes until the expiration provided in the response.
+
+#### Protecting Against Bad Actors
+
+To protect against bad actor clients reserving all available captial without following through with the requested trades, servers should only accept requests for which the following is true:
+
+1. The Stellar account authenticated via SEP-10 is owned by a known and trusted entity, or
+2. The owner of the Stellar account has been properly KYC'ed, either via [SEP-12](sep-0012.md) or some other mechanism.
```

Comment:
🤔 These two points seem like the same thing to me. If the account is owned by a known entity then KYC has been performed, because KYC is know-your-customer.

Reply:
I can see the possibility for confusion, I was thinking about it in the context of SEP-31 v. SEP-6.

In SEP-31, the owner of the Stellar account is a business that would have an agreement with the quote-providing anchor and would be liable for misuse of the service

In SEP-6, the owner of a Stellar account an individual (unless its a custodial service, in which case an agreement could be made with the custodian) who couldn't realistically be held liable but would have to provide personal identification info.

I tried making this business vs. individual distinction with the word `entity` but maybe thats not clear enough.